app/models/UserModel.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Models;
+use App\Models\Repositories\UserRepository;
+
+class UserModel
+{  
+    public $userData = [
+        'name' => '',
+        'surname' => '',
+        'email' => '',
+        'gender' => '',
+        'status' => '',
+    ];
+    public $error = '';
+    public $userExists = false;
+    
+    public function setData($data)
+    {
+        foreach($data as $key => $value) {
+            if(array_key_exists($key, $this->userData)) {
+                $this->userData[$key] = $value;
+            }
+        }
+    }
+
+    public function validation(): string
+    {
+        if ($this->userExists != false) {
+            return $this->error = 'User is already exist';
+        }
```

Comment:
Could: In the block below, each validator call overrides the error message, try something like this:
```
foreach(['surnameValidation', 'nameValidation', ..., 'statusValidation'] as $validator)  {
$this->$validator(); 
if($this->error) break;
}
```